### PR TITLE
Issue-862: Fix pagination and datatable on dictionary menu  page

### PR DESCRIPTION
**Pull request by mherman22**
# Pull Requests Requirements

- [x] The PR title includes a brief description of the work done, including the Issue number if applicable.
- [x] The PR includes a video showing the changes for the work done.
- [x] The PR title follows conventional commit label standards.
- [x] The changes confirm to the OpenElis Global x3 Styleguide and design documentation.
- [x] The changes include tests or are validated by existing tests.
- [x] I have read and agree to the Contributing Guidelines of this project.

## Screenshots

[Add relevant screenshots here if applicable]

## Related Issue

[Add a link to the related issue or mention it here if applicable]

## Other

[Add any additional information or notes here]

### CODE REVIEW

File: frontend/src/components/admin/menu/DictionaryManagement.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ function DictionaryManagement() {
       <PageBreadCrumb
         breadcrumbs={[
           { label: "home.label", link: "/" },
-          { label: "dictionary.label.modify", link: "/DictionaryManagement" },
+          { label: "dictionary.label.modify", link: "MasterListsPage#DictionaryMenu" },
```
Comment (mozzy11) [inline frontend/src/components/admin/menu/DictionaryManagement.js:167]: This change isnt needed

**Comment by mherman22** [comment]
> Hey @mherman22 for the modification part I see that you’ve already set up the REST controller for the dictionary .Therefore, you should be able to use the `/rest/Dictionary `endpoint.And also Please ensure to include the specific list’s ID and startingRecNo as a parameter in your POST request.
> 
> So basically the endpoint would be something like` rest/Dictionary?ID=1&startingRecNo=1`

Tried that already.  Anyway lemi give it another shot later. Lemi take a breather for now

File: frontend/src/components/admin/menu/DictionaryManagement.js [hunk 2]
```diff
@@ -200,56 +392,95 @@ function DictionaryManagement() {
...
+                sm={4}
+                style={{
+                  display: "flex",
+                  flexDirection: "column",
+                  alignItems: "center",
+                  gap: "10px",
+                }}
+              >
+                <Link>
+                  Showing {toRecordCount} - {fromRecordCount} of{" "}
```
Comment (mherman22) [inline frontend/src/components/admin/menu/DictionaryManagement.js:464]: The button is disabled if paging is 1 and `startingRecNo` is less than or equal to 21.
The button is also disabled if `startingRecNo` is 1 or less.

File: frontend/src/components/admin/menu/CommonProperties.js [hunk 3]
```diff
@@ -61,105 +61,100 @@ export const CommonProperties = () => {
   };
 
   return (
```
Comment (mozzy11) [inline frontend/src/components/admin/menu/CommonProperties.js:63]: are these changes part of this PR ??

**Pull request merged**